libs/net/models/Areas/Subscriber/Models/Report/ReportInstanceContentModel.cs

```diff
@@ -64,7 +64,7 @@ public ReportInstanceContentModel(Entities.ReportInstanceContent entity) : base(
         return new Entities.ReportInstanceContent(model.InstanceId, model.ContentId, model.SectionName)
         {
             SortOrder = model.SortOrder,
-            Content = model.Content != null ? (Entities.Content)model.Content : null,
+            // Content = model.Content != null ? (Entities.Content)model.Content : null,
```

Comment:
I don't think we need to deserialize content with this endpoint model.  However, it's possible I've introduced a regression isssue